requirements.txt

```diff
@@ -26,3 +26,4 @@ sentencepiece==0.1.95
 # for code format
```

Comment:
We don't need this, it's already there as a pre-commit hook in the .pre-commit-config file.

Reply:
tht was in master...my code didnt add it